compose.yaml

```diff
@@ -39,17 +39,16 @@ services:
     image : docker.io/vxingest/development:dev
     volumes:
       - ${data}:/opt/data
-      - /public:/public
+      - ${public}:/public
     secrets:
       - CREDENTIALS_FILE
-    command: ./scripts/VXingest_utilities/run-ingest.sh -c /run/secrets/CREDENTIALS_FILE -l ${LOGDIR} -o ${OUTDIR} -m ${METRICSDIR} -x ${TRANSFERDIR}
```

Comment:
In the future it'd be nice to switch back to having the flags in the compose file, with the [values configured by the `.env` file that Docker Compose automatically looks for](https://docs.docker.com/compose/environment-variables/set-environment-variables/#substitute-with-an-env-file) so that future us won't need to look up the flags.

Reply:
This is changed a little now. I think the CREDENTIALS _FILE always needs to be in the same place. I din't want users putting them in some shared directory so I enforced them being in home, and the other parts are all changed now. They aren't env vars anymore. Perhaps we need a new PR?

Reply:
I think this PR is good. If we wanted to do that change, I do think it'd be in a new PR. 

The `.env` file would have values to where local system resources are located - i.e. the credentials file, data dir, and public dir. The `-c /run/secrets/CREDENTIALS_FILE` flag (and others in the `compose.yml` file) would remain hardcoded since it's an implementation detail inside the container that the user doesn't have control of.

E.g. the invocation would change from: 
```
data=/data-ingest/data \
public=/public \
docker compose run ingest ./scripts/VXingest_utilities/run-ingest.sh -c /run/secrets/CREDENTIALS_FILE -o /opt/data/test/outdir -l /opt/data/test/logs -m /opt/data/test/metrics -x /opt/data/test/xfer
```

to :
```
docker compose run ingest
```

With the following files/changes:
A new `.env` file:
```env
LOCAL_SECRETS=$HOME/credentials
LOCAL_DATA=/local/path/to/data
LOCAL_PUBLIC=/local/path/to/public
```

And changes to the `compose.yml` like:
```yaml
...
secrets:
  CREDENTIALS_FILE:
    file: ${LOCAL_SECRETS}
...
 ingest:
    image : docker.io/vxingest/development:dev
    volumes:
      - ${LOCAL_DATA}:/opt/data
      - ${LOCAL_PUBLIC}:/public
    secrets:
      - CREDENTIALS_FILE
    command: ./scripts/VXingest_utilities/run-ingest.sh -c /run/secrets/CREDENTIALS_FILE -o /opt/data/test/outdir -l /opt/data/test/logs -m /opt/data/test/metrics -x /opt/data/test/xfer
```